pkg/connect/client/datastore/v1alpha1/datastore.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2025 Cofide Limited.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	"context"
+
+	datastorev1alpha1 "github.com/cofide/cofide-api-sdk/gen/go/proto/connect/datastore_service/v1alpha1"
+	"google.golang.org/grpc"
+)
+
+// DataStoreClient is an interface for a client for the v1alpha1 version of the Connect DataStoreService.
+type DataStoreClient interface {
+	CountAttestedNodes(ctx context.Context, req *datastorev1alpha1.CountAttestedNodesRequest) (*datastorev1alpha1.CountAttestedNodesResponse, error)
```

Comment:
This does break the existing SDK client pattern a bit by not unwrapping the request and response types. Perhaps less important as it's not a user-facing API.

Reply:
Yeah, that's what I was thinking, and we can always evolve this later.